util/Conventional_Monitor/data_extract/ush/ConMon_DE.sh

```diff
@@ -229,7 +229,7 @@ if [ -s $cnvstat  -a -s $pgrbf00 -a -s $pgrbf06 ]; then
       fi
 
       if [[ $MY_MACHINE = "wcoss_d" || $MY_MACHINE = "wcoss_c" ]]; then
-        $SUB -q $JOB_QUEUE -P $PROJECT -o ${logfile} -M 900 \
+        $SUB -q $JOB_QUEUE -P $PROJECT -o ${logfile} -M 1100 \
```

Comment:
This is a straightforward and necessary change.   Would it be good to increase the memory limit even more?   RadMon jobs specify `-M 5000`  I agree that we don't want to request too much memory.  At the same time, we don't want to keep incrementally increasing the memory limit.  Just a thought.